soc/arm/st_stm32/stm32f1/Kconfig.defconfig.stm32f100xx

```diff
@@ -0,0 +1,51 @@
+# ST Microelectronics STM32F100XX MCU
+
+# Copyright (c) 2020, Jonas Eriksson, Up to Code AB
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_STM32F100XB
+
+config SOC
+	default "stm32f100xb"
+
+config NUM_IRQS
+	default 41
+
+endif # SOC_STM32F100XB
+
+if SOC_STM32F100XE
+
+config SOC
+	default "stm32f100xe"
+
+config NUM_IRQS
+	default 60
+
+endif # SOC_STM32F100XE
+
+if GPIO_STM32
+
+# GPIO ports A, B and C are set in ../common/Kconfig.defconfig.series
+# GPIO port D are set in ./Kconfig.defconfig.series
+
+if SOC_STM32F100VX
+
+config GPIO_STM32_PORTE
+	default y
+
+endif # SOC_STM32F100VX
+
+if SOC_STM32F100ZX
+
+config GPIO_STM32_PORTE
+	default y
+
+config GPIO_STM32_PORTF
+	default y
+
+config GPIO_STM32_PORTG
+	default y
+
+endif # SOC_STM32F100ZX
+
+endif # GPIO_STM32
```

Comment:
Let's remove these for now.

Reply:
Alright. Do you want them in some other way for the SoC, or is this best done in board configurations? ..or perhaps left out awaiting the use case.

Reply:
Let's keep it minimum for now

Reply:
Alright